tests/containers/docker.pm

```diff
@@ -47,8 +47,11 @@ sub run {
 
     if ($self->firewall() eq 'firewalld') {
         # on publiccloud we need to install firewalld first
-        install_and_start_firewalld() if (is_publiccloud || is_jeos);
-        check_docker_firewall();
+        zypper_call('install firewalld') if (is_publiccloud || is_jeos);
+        systemctl('restart firewalld');
+        systemctl('restart docker');
+        $stop_firewall = 1;
+        check_containers_firewall($engine);
     }
```

Comment:
I'd add at least a `record_info` that no firewall test is performed. Swallowing tests silently is dangerous.

Reply:
This is done right on top of `check_containers_firewall()` - feel free if you disagree ;)